skimage/feature/_hog.py

```diff
@@ -124,6 +151,15 @@ def hog(image, orientations=9, pixels_per_cell=(8, 8), cells_per_block=(3, 3),
 
     image = np.atleast_2d(image)
 
+    if multichannel is None:
+        multichannel = (image.ndim == 3)
```

Comment:
What happens when alpha images land here?

Reply:
AFAIK, most of our functions don't work well with RGBA (only some filters are implemented in an appropriate way).

Reply:
In any case, this line was proposed by @jni, so, please, consider blaming him :smile: . There was no guessing code in the previous version of this PR.

Reply:
This would still work totally fine with an RGBA image.

Reply:
And the result will be identical to using an RGBA image with `multichannel=True`, as you would. Namely, the gradients in the alpha layer will "count". Dunno if that's what you want, but this line isn't the problem.

Reply:
Yes, of course, you are right. I'll try to find some info on HOG for RGBA. In the original paper there weren't any hints on this case.

Reply:
I misread; thought this checked the final dimension, but see it only checks the number of dimensions.